db/migrate/20201006172019_add_ancillary_metadata_to_portfolio_item.rb

```diff
@@ -0,0 +1,5 @@
+class AddAncillaryMetadataToPortfolioItem < ActiveRecord::Migration[5.2]
+  def change
+    PortfolioItem.all.each(&:update_metadata)
+  end
```

Comment:
How does this work for a down migration? I don't think rails knows how to handle this statement for down vs. up, so I would imagine it does the same thing, which doesn't seem like what we want to happen. Do we need to remove data from the database for the down migration?